modules/oadp-release-notes-1-1-6.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-1-6_{context}"]
+= OADP 1.1.6 release notes
+
+The OADP 1.1.6 release notes lists any new features, resolved issues and bugs, and known issues.
+
```

Comment:
@kaovilai && @weshayutin - are there any technical features you want to highlight?
Do you want me to include the version of Velero explicitly mentioned here?